import "trident/rpc_option.proto";

package trident.test;

option cc_generic_services = true;

message EchoRequest {
    required string message = 1;
}

message EchoResponse {
    required string message = 1;
}

service EchoServer {
    rpc Echo(EchoRequest) returns(EchoResponse) {
        option (trident.request_compress_type) = CompressTypeSnappy;
        option (trident.response_compress_type) = CompressTypeSnappy;
    }
}
